simple_network_sim/network_of_populations.py

```diff
@@ -374,31 +374,31 @@ def doInternalInfectionProcess(
     :return: The number of new infections stratified by age.
     :rtype: A dictionary of {age: number of new infections}
     """
-    newInfectedsByAge = {}
+    newInfectedByAge = {}
     for age in ageMixingInfectionMatrix:
-        newInfectedsByAge[age] = 0
+        newInfectedByAge[age] = 0
 
         numSuscept = getSusceptibles(age, currentInternalStateDict)
-        if numSuscept>0:
-            numInfectiousContactsFromAges = {}
+        if numSuscept > 0:
             totalNewInfectionContacts = 0
             for ageInf in ageMixingInfectionMatrix[age]:
                 totalInfectious = getInfectious(ageInf, currentInternalStateDict, infectiousStates)
                 # TODO: Make sure this is not implemented in a slow way anymore after https://github.com/ScottishCovidResponse/SCRCIssueTracking/issues/273
-                numInfectiousContactsFromAges[ageInf] = totalInfectious*ageMixingInfectionMatrix[ageInf][age]
-                totalNewInfectionContacts = totalNewInfectionContacts + numInfectiousContactsFromAges[ageInf]
-#      Now, given that we expect totalNewInfectionContacts infectious contacts into our age category, how much overlap do we expect?
-#       and how many are with susceptible individuals? 
+                totalNewInfectionContacts += totalInfectious * ageMixingInfectionMatrix[ageInf][age]
```

Comment:
I think you can just use `getInfectious(ageInf, currentInternalStateDict, infectiousStates)` here and avoid the totalInfectious variable altogether.

Reply:
thanks, fixed